sessions/react-component-testing/demo-end/components/Movies/Movies.test.js

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Movies from ".";
+
+const initialMovies = [
+  {
+    id: "0",
+    name: "The Incredible Hulk",
+    isLiked: false,
+  },
+  {
+    id: "1",
+    name: "Spiderman 1-25",
+    isLiked: false,
+  },
+  {
+    id: "2",
+    name: "Lord of the Rings",
+    isLiked: true,
+  },
+];
+
+test("renders the initialMovies", async () => {
+  render(<Movies initialMovies={initialMovies} />);
+  expect(
+    screen.getByRole("heading", { name: "The Incredible Hulk" })
+  ).toBeInTheDocument();
+  expect(
+    screen.getByRole("heading", { name: "Spiderman 1-25" })
+  ).toBeInTheDocument();
+  expect(
+    screen.getByRole("heading", { name: "Lord of the Rings" })
+  ).toBeInTheDocument();
+});
```

Comment:
No need for `async`. We should use the same code as in the session guide.

```suggestion
test("renders the initialMovies", () => {
  render(<Movies initialMovies={initialMovies} />);
  const hulkHeading = screen.getByRole("heading", {
    name: "The Incredible Hulk",
  });
  expect(hulkHeading).toBeInTheDocument();

  const spidermanHeading = screen.getByRole("heading", {
    name: "Spiderman 1-25",
  });
  expect(spidermanHeading).toBeInTheDocument();

  const lordOfTheRingsHeading = screen.getByRole("heading", {
    name: "Lord of the Rings",
  });
  expect(lordOfTheRingsHeading).toBeInTheDocument();
});
```